packages/features/form-builder/FormBuilder.tsx

```diff
@@ -551,7 +551,19 @@ function FieldLabel({ field }: { field: RhfFormField }) {
       "Field has `variantsConfig` but no `defaultVariant`" + JSON.stringify(fieldTypeConfigVariantsConfig)
```

Comment:
@MehulZR I still see some unrelated changes here in this file related to #10390. Let's remove those first as I mentioned earlier.